pkg/machines/machines.less

```diff
@@ -244,6 +244,24 @@ td > .utilization-bar-pf {
     background-color: var(--color-ct-list-critical-bg) !important;  /* keep red background when expanded */
 }
 
+// PF4 override: fix toggle expandable; it has a light grey background otherwise
+.pf-c-expandable__toggle {
+    // Mimic a link's focus (to fix Firefox's lack of focus)
+    &:hover,
+    &:focus {
+        outline: none;
+
+        > span {
+            text-decoration: underline;
+        }
+    }
+}
+
+// PF4 override: rely on ct-form for spacing; remove local margin
+.ct-form > .pf-c-expandable .pf-c-expandable__content {
+    margin-top: 0;
+}
+
```

Comment:
These can go if the expander is dropped. Of course, if you want to keep it, but just open it by default, that's also fine. I don't have a strong opinion about it, maybe @garrett has.